nodejs/dist/uws.js

```diff
@@ -25,17 +29,22 @@ class Socket {
      * @param {Function} f
      */
     on(eventName, f) {
+        if (typeof f !== 'function') {
+            throw new TypeError('Event listener must be a function');
+        }
```

Comment:
I know, its probably not going to make any difference but it's just the whole thing with JavaScript making me so angry. The language is just pure crap and broken by design, making the programmer ignore types, but then also having to write checks that the compiler should normally have done had it been a sane language. #flame